docs/howto/writing-migrations.txt

```diff
@@ -326,3 +326,35 @@ If you want to change an unmanaged model (:attr:`managed=False
 ``managed=False`` and generate a migration before making other schema-related
 changes to the model, since schema changes that appear in the migration that
 contains the operation to change ``Meta.managed`` may not be applied.
+
+Disconnecting a subclass from a concrete parent model
+=====================================================
+
+If you have a model ``Parent`` and a subclass ``Child`` and you want to make
+the ``Child`` model independent you need to write a custom migration. The key
+parts are ``AlterModelBases`` to tell django what the new model hierarchy is
+and ``AlterField`` to convert the implicit ``OneToOneField`` into an implicit
+``AutoField`` primary key. The ``RenameField` is useful to avoid needing to add
+the ``parent_ptr`` field explicitly to the updated ``Child`` model.
```

Comment:
I'm not sure if starting from an example is a good way to describe use cases :thinking: `AlterModelBases` should be used also in other cases e.g. renaming parent.

Maybe:
```
If you want to change parent models with
:ref:`multi-table inheritance <multi-table-inheritance>` you need to write a
custom migration e.g. to rename a parent model or make a child model
independent. ...